keras_cv/layers/object_detection/nms_prediction_decoder.py

```diff
@@ -15,115 +15,98 @@
 import tensorflow as tf
 
 from keras_cv import bounding_box
-from keras_cv import layers as cv_layers
 
 
-class NmsPredictionDecoder(tf.keras.layers.Layer):
+# TODO(tanzhenyu): provide a TPU compatible NMS decoder.
```

Comment:
Can you add an expected failing test with jit_compile=True?

Reply:
What's expected failing test? Do you have an example?

Reply:
`@unittest.expectedFailure` with the NMSDecoder with `jit_compile=True`